src/components/basic/IconButton.md

```diff
@@ -0,0 +1,21 @@
+```jsx
+import IconButton from './IconButton';
+<IconButton
+icon = "fa fa-users" 
+type="Default"
+size="large"
+description="Show Icon"
+text="TRASH AND RECYCLING SCHEDULE"
```

Comment:
This text should be use regular casing, the css will handle the uppercase piece, this offers us the flexabilty to change the way it's displayed.